metadata/report/etcd/report_test.go

```diff
@@ -94,8 +94,14 @@ func TestEtcdMetadataReport_CRUD(t *testing.T) {
 	err = metadataReport.SaveSubscribedData(subMi, string(urls))
 	assert.Nil(t, err)
 
+	metadataInfo := common.NewMetadataInfWithApp(subMi.Application)
 	err = metadataReport.RemoveServiceMetadata(serviceMi)
 	assert.Nil(t, err)
+	err = metadataReport.PublishAppMetadata(subMi, metadataInfo)
+	assert.Nil(t, err)
+
+	_, err = metadataReport.GetAppMetadata(subMi)
```

Comment:
Why don't check the return value of `GetAppMetadata` here?

Reply:
Yes, It should be...